tox.ini

```diff
@@ -24,8 +24,14 @@ norecursedirs = .* docs requirements
 [testenv]
 deps =
     django18: Django>=1.8,<1.9
+    django18: djangorestframework>=3.2.3,<3.3.0
+    django18: django-filter>=0.11.0,<1.0.0
     django19: Django>=1.9,<1.10
+    django19: djangorestframework>=3.2.3,<3.3.0
+    django19: django-filter>=0.11.0,<1.0.0
```

Comment:
I know I'm late to the party, but you can consolidate this: `django1{8,9}: djangorestframework>=3.2.3, <3.3.0`